.github/workflows/release.yml

```diff
@@ -0,0 +1,48 @@
+---
+name: release
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+      packages: write
+
+    steps:
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@gh_actions
```

Comment:
I'd prefer to use the stable release, `gh_actions` is not the correct one but `current`. This will help us to use the stable version, rather than the latest changes

Reply:
```suggestion
      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
```

Reply:
Alright 👍 

The only issue I see is, when an action that is used in the release workflow is actually buggy, how do you release the fix for it?  I guess, this is an edge case and I get your point.

Reply:
`current` is a tag, so we can manually release the tag if needed in the case of that particular corner-case, does it sound like a good plan? 

I like the idea of dog-fooding our actions, but better by doing so in the CI workflows instead of the release ones. We can try this approach and if needed we can change it, what do you think?

Reply:
Just thinking out loud here:

Using the tag, might create an edge-case where you have to do manual intervention if necessary.

Using the local action, could be unstable. So you are forced to fix the used action and trigger another release. Which is basically blocking the release.

Just my 2 cents.

But I'm fine with both.. since both of ups and downsides.